test/functional/feature_asset_locks.py

```diff
@@ -330,6 +330,16 @@ def run_test(self):
 
         txid = self.send_tx(asset_unlock_tx)
         assert "assetUnlockTx" in node.getrawtransaction(txid, 1)
+
+        indexes_statuses = self.nodes[0].getassetunlockstatuses(["101", "102", "300"])
+        self.log.info(f'{indexes_statuses}')
+        assert any('101' in d for d in indexes_statuses)
```

Comment:
why not simple `assert_equal(indexes_statuses, ["mempooled", None, None])` ?

Reply:
Well, I want to make sure that '101' is also returned.